static/src/javascripts/types/global.d.ts

```diff
@@ -65,6 +65,10 @@ interface CommercialPageConfig {
 interface Config {
 	page: PageConfig;
 	switches: Record<string, boolean | undefined>;
+	tests?: {
+		testName: string;
+		groupName: string;
+	};
```

Comment:
I don’t this this type is accurate. Based on sample data, `window.guardian.config.tests
` looks more like this:

```ts
{
    interactiveLibrarianVariant: "variant", // from the live website
    someTestControl: "control", // IIRC
    anotherTestVariant: "variant", // IIRC
}
```

Therefore, the type would be closer to:

```suggestion
	tests?: Record<string, 'control' | 'variant'>;
```

Reply:
Yes, good call!